containers/keycloak-ers/handlers/entityResolution.go

```diff
@@ -0,0 +1,237 @@
+package handlers
+
+import (
+	ctx "context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/Nerzal/gocloak/v11"
+	"go.opentelemetry.io/otel"
+	"go.uber.org/zap"
+)
+
+var tracer = otel.Tracer("handlers")
+
+// EntityResolution Model
+type EntityResolution struct {
+	Identifier        string   `json:"identifier" example:"bob@sample.org"`
+	EntityIdentifiers []string `json:"entity_ids" example:"XUX-ASDF-ASDFSD"`
```

Comment:
```suggestion
	EntityIdentifiers []string `json:"entity_ids" example:"XUX-ASDF-ASDFSD"`
	AttributeType     string             `json:"type" example:"email"`
```

I would like type moved so it's on the same level as the resolution itself, so you could respond with e.g. a mixture of types if you got asked to resolve a mixture of types.